### ISSUE TITLE
Can't change the lineColor in a timeline diagram

### ISSUE THREAD

**Issue by blackmad** [mermaid-js/mermaid#4263]
### Description

I'm struggling to figure out if it's at all possible to change the line color of timeline diagrams.

I've read the docs for timeline https://mermaid.js.org/syntax/timeline.html which doesn't talk about line color, and also https://mermaid.js.org/config/theming.html#theme-variables-reference-table 

Is the line color configurable? how can I figure out what variable controls it?

I'm trying lineColor and actorLineColor, but neither seems to do it.

### Steps to reproduce

https://mermaid.live/edit#pako:eNplUcFqwzAM_RWhEXzJIZSdct3YVmhhUNgpFyWWM1PHGo4zKCH_PqVjXbvJF_u9pydLmrETy1gjaBTF7KPPNcxggvQ7_uRgajCW26k3JZj8zgOvSEsjG_hB3ih5agOPSs1NhJswwUd-kCBpTbxzrtIw5T8ZdVnS7kZbVavaXKTLeori8s5-4NX91yv7HBhe_KheJxAHB-k8Bdiz9QSvgbKTNFzX3lTVBmrQwke22_iHulfqiTpuRY56fRbpg5bDEgdOA3mrkzt33ODtJBrUUSzlFbUiDVp2NIXcYBMXdaEpy-EUO6xzmrjE6cNS5kdPfaIBa0dhVFQ_r_3svxd13tfyBb35gaY(

### Code Sample

```text
%%{init: { 'logLevel': 'debug', 'theme': 'base' , 'themeVariables': {
              'lineColor': '#ff0000',
              'actorLineColor': '#00ff00'
       } } }%%
       timeline
        title History of Social Media Platform
          2002 : LinkedIn
          2004 : Facebook : Google
```
```


### Setup

- Mermaid version: 10.0.2
- Browser and Version: Chrome

### Additional Context

_No response_

**Comment by jgreywolf** [comment]
@blackmad No, you are not able to change the line color in timeline diagram at this time.  We will consider it for a future version
